samples/boards/nxp/mimxrt1170_evk_cm7/magic_addr/sample.yaml

```diff
@@ -9,5 +9,5 @@ common:
     - pytest
 tests:
   sample.boards.mimxrt1170_evk.magic_addr:
-    filter: CONFIG_MEMC_NXP_FLEXRAM
+    filter: CONFIG_MEMC_NXP_FLEXRAM and dt_node_prop_enabled("flexram", "flexram,has-magic-addr")
```

Comment:
do you still need `DT_HAS_NXP_FLEXRAM_ENABLED` here? this is enabled depending on DTS already.

Reply:
yes, we can remove the CONFIG_MEMC_NXP_FLEXRAM updated